src/msg/async/rdma/RDMAConnectedSocketImpl.cc

```diff
@@ -71,6 +72,16 @@ RDMAConnTCP::RDMAConnTCP(CephContext *cct, RDMAConnectedSocketImpl *sock,
   my_msg.peer_qpn = 0;
   my_msg.gid = ibdev->get_gid(ibport);
   socket->register_qp(qp);
+
+  if (_info) {
```

Comment:
_info exists only for server sockets.
asking if (_info), is actually asking if is_server....